user/views.py

```diff
@@ -60,3 +59,21 @@ def post(self,request):
         except KeyError :
             return JsonResponse({"message": "KEY_ERROR"}, status=400)
 
+class SigninView(View):
+    def POST(self,request):
```

Comment:
handler method인 post를 소문자로 만들어주셔야 urls.py에서 `SigninView.as_view()`함수를 호출할 때, http POST 요청이 들어오면 SigninView 클래스의 post method를 실행시켜줍니다~

```suggestion
    def post(self, request):
```